source/CargoHold.cpp

```diff
@@ -602,15 +601,15 @@ int CargoHold::IllegalCargoFine(const Government *government) const
 			return -1;
 		if(fine < 0)
 			return fine;
-		totalFine = max(totalFine, fine / 2);
+		totalFine += (fine / 2) * it.second;
```

Comment:
Ok to make the fines stack but maybe use a sqrt or smth?
Or maybe we should just rebalancr fines afterwards, but 450M caus of nerve gas is going to hurt (which is good), possibly too much

Reply:
Yeah definitely